_config/extensions.yml

```diff
@@ -4,3 +4,9 @@ name: extensible-search-extensions
 SearchForm:
   extensions:
     - 'SearchFormResultsExtension'
+ExtensibleSearchSuggestion:
```

Comment:
Agree with this. The configuration should be defined from the project, and not here.